### PR TITLE
feat: create page for job openings

### PR DESCRIPTION
**Pull request by jfcurran**
This pull-request is still in progress.

This PR aims to create a jobs page that contains the following:

- `hero.tsx` a hero with image
- `new` a list of job openings, generated using a graphQL query, listing out available job openings - with each opening being a markdown file that lives in `./jobs/job.md`
  - .mds used for job openings must contain position, starting date, location/remote, and more (please add) 
- `features.tsx` a component that displays our company values in succinct and comprehensible fashion 
- `features.tsx` a component that displays four benefits of working for us
- `quotes.tsx` TBD a component featuring quotes from team members
- `faq.tsx` component with frequently asked questions
- `quickstart.tsx` a component that allows for contact. 
  - Are you looking for other ways to work with us? Get in touch!
- `footer.tsx`

### CODE REVIEW

**Comment by aeneasr** [comment]
I have another PR with important changes that will cause a lot of conflicts. Can we get this merged? Otherwise I'll go ahead with the next changes.

**Comment by jfcurran** [comment]
@vinckr This page is looking good for now - is the content final? if yes, please give it a pass and check for typos and alike, thank you. I didn't find any, but I saw the writing of Open Source and open source to be inconsistent

File: gatsby-node.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const webpack = require('webpack')
 
 const trimLeft = (s, charlist) => {
   if (charlist === undefined) {
-    charlist = '\s'
+    charlist = 's'
```
Comment (aeneasr) [inline gatsby-node.js:7]: We reverted this, now it's back. Please revert it again.

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: gatsby-node.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const webpack = require('webpack')
 
 const trimLeft = (s, charlist) => {
   if (charlist === undefined) {
-    charlist = '\s'
+    charlist = '/s'
```
Comment (aeneasr) [inline gatsby-node.js:7]: Cmon...

```suggestion
    charlist = '\s'
```

File: src/components/layouts/features/features-content.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+
+import Button from '../../freestanding/button/button'
+import IconWrapper from '../../freestanding/icon/icon-wrapper'
+
+const ArrowRightThemed = (
+  <IconWrapper color={'themed-primary'} icon={'ArrowRightBold'} size={'16'} />
```
Comment (aeneasr) [inline src/components/layouts/features/features-content.tsx:7]: We deprecated IconWrapper, please use `<i>`

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by aeneasr** [comment]
Please address the merge conflicts with blog-hero

File: gatsby-node.js (hunk 1 above)
Comment (aeneasr) [reply gatsby-node.js:7]: Dude, please, it's still not correct. Just accept this patch, please:

```suggestion
    charlist = '\s'
```

**Comment by aeneasr** [comment]
Shipped 🎉

**Comment by jfcurran** [comment]
this better not be the Ever Given
image

**Pull request merged**